e2e/conf/protractor.base.conf.js

```diff
@@ -18,7 +18,10 @@ exports.config = {
     testPath +  '/**/*.feature'
   ],
   capabilities: {
-    'browserName': 'chrome'
+    'browserName': 'chrome',
+    'chromeOptions': {
+      'args': ['--no-sandbox']
+    }
```

Comment:
is this option common to all protractor configurations ?
If it's just for `yarn e2e:ipaas-qe`, we might move `chromeOptions` to `e2e/conf/protractor.ipaas-qe.conf.js`